dependencies/Dockerfile

```
@@ -2,25 +1,62 @@
+ARG TARGETPLATFORM
+SHELL ["/bin/bash", "-euo", "pipefail", "-c"]
+COPY --from=version-detector /versions.env /versions.env
+COPY --chmod=0755 prepare.sh /tmp/
+RUN /tmp/prepare.sh \
+    && mkdir -p /output/bin /output/lib /output/include /output/pkgconfig /output/etc /output/deb
+
+FROM base-prep AS build-libraw
+ARG LIBRAW_VERSION
+WORKDIR /tmp/build-libraw
+COPY --chmod=0755 build_libraw.sh .
+RUN --mount=type=secret,id=github_token \
```

Hadolint (2.12.0)

[error] Unexpected start of line. If breaking lines, |/||/&& should be at the end of the previous one.

(SC1133)